src/test/javascript/spec/component/shared/metis/post/post.component.spec.ts

```diff
@@ -27,8 +29,7 @@ describe('PostComponent', () => {
     beforeEach(() => {
         return TestBed.configureTestingModule({
             imports: [],
-            declarations: [PostComponent, MockPipe(HtmlForMarkdownPipe)],
-            schemas: [NO_ERRORS_SCHEMA],
+            declarations: [PostComponent, MockPipe(HtmlForMarkdownPipe), MockComponent(PostHeaderComponent), MockComponent(PostFooterComponent)],
             providers: [],
```

Comment:
Bonus points if you remove the lines with empty arrays `[]`, as they shouldn't be needed.